Dockerfile

```diff
@@ -85,16 +85,21 @@ COPY data /adaguc/adaguc-server-master/data
 COPY python /adaguc/adaguc-server-master/python
 
 ######### Third stage, test ############
-FROM base AS test
+# To run the tests against a postgres db, see docs/test_postgesql.md
+FROM base as test
+
+ENV TEST_IN_CONTAINER 1
 
 COPY requirements-dev.txt /adaguc/adaguc-server-master/requirements-dev.txt
 RUN pip install --no-cache-dir -r requirements-dev.txt
 
 COPY tests /adaguc/adaguc-server-master/tests
 COPY runtests.sh /adaguc/adaguc-server-master/runtests.sh
+COPY runtests_psql.sh /adaguc/adaguc-server-master/runtests_psql.sh
 
-# Run adaguc-server functional and regression tests
+# Run adaguc-server functional and regression tests. See also `./doc/developing/testing.md`
 RUN bash runtests.sh
+# RUN bash runtests_psql.sh
```

Comment:
Curious to know if this could work in github actions? That would be nice! In the future we could get rid of sqlite completely.